### DISCUSSION TITLE
Is it possible to consume messages from offloaded ledgers?

### DISCUSSION THREAD

**Discussion by pckhoi** [apache/pulsar#22113] [Q&A] [answered]
It is said in the Tiered Storage Overview that "After offloading ledgers to long-term storage, you can still query data in the offloaded ledgers with Pulsar SQL." Does that mean that regular consumers will not be able to consume from those ledgers?

**Comment by jak78** [comment, marked as answer]
Hi @pckhoi! Regular consumers can still consume messages from offloaded ledgers.
Feel free to check the [Tiered Storage docs](https://pulsar.apache.org/docs/3.2.x/tiered-storage-overview/):
> Pulsar's Tiered Storage feature allows older backlog data to be moved from BookKeeper to long-term and cheaper storage, while still allowing clients to access the backlog as if nothing has changed.

Best,

**Reply by pckhoi** [reply]
Thanks for the clarification! I didn't see an example so I needed a clear answer.

**Reply by jak78** [reply]
I'm glad that clarified things. I just created a [PR](https://github.com/apache/pulsar-site/pull/810) to improve the doc thanks to your feedback.
